lib/src/react16_suggestors/dependency_override_updater.dart

```diff
@@ -85,17 +121,64 @@ class DependencyOverrideUpdater implements Suggestor {
   bool shouldSkip(_) => false;
 }
 
-bool containsDependencyOverride({String dependency, String fileContent}) {
-  final regexString = r'''^\s*\w{0,4}\s*:\s*[\s\S]*(.com){0,1}.\w*\/''' +
-      dependency +
-      r'''[\s\S]*\n{0,1}$''';
+bool fileContainsDependencyOverride({String dependency, YamlMap yamlContent}) {
```

Comment:
These arguments should be required (either positional or with the `@required` annotation)